src/kolibri_android/android_utils.py

```diff
@@ -164,6 +164,20 @@ def get_home_folder():
     return os.path.join(kolibri_home_file.toString(), "KOLIBRI_DATA")
 
 
+def get_initial_content_pack_id():
+    preferences = get_preferences()
+    pack_id = preferences.getString("initial_content_pack_id", None)
+    logger.debug("Initial content pack ID: %s", pack_id)
+    return pack_id
+
+
+def set_initial_content_pack_id(pack_id):
+    logger.debug("Setting initial content pack ID: %s", pack_id)
+    editor = get_preferences().edit()
+    editor.putString("initial_content_pack_id", pack_id)
+    editor.commit()
+
+
```

Comment:
This setter isn't being used. Do we expect it will be useful?

Reply:
Ah true the I ended up storing the pack ID in the Java code. Thanks! removing.